src/error.rs

```diff
@@ -156,6 +167,20 @@ pub enum ErrorKind {
     /// defined for a field, but the corresponding resolver is not provided.
     ResolverNotFound(String, String),
 
+    /// This error is returned if the root node could not be determined from the schema.
+    /// Note: This error should never be thrown. This is a critical error. If you see it,
+    /// please report it to the warpgrapher team.
+    MissingRootNode,
+
+    /// This error is returned if the database pool was not created.
+    /// Note: This error should never be thrown. This is a critical error. If you see it,
+    /// please report it to the warpgrapher team.
+    MissingDatabasePool,
+
+    /// Returned by the `From` implementation for a `serde_json::error::Error` object.
+    /// Conversion from `serde_json::error::Error`.
+    SerdeJsonError(String),
```

Comment:
This tells me about the mechanics of the source of the error, but not really what causes it.  Why would someone get this from Warpgrapher?

Reply:
Removed this error